src/tests/ftest/server/metadata.py

```diff
@@ -27,9 +27,15 @@
 import traceback
 import uuid
 import threading
-import Queue
 import avocado
 
+try:
+    # python 3.x
+    import queue as queue
+except ImportError:
+    # python 2.7
+    import Queue as queue
```

Comment:
I'm surprised this didn't raise a pylint error itself, as pylint3 presumably thinks that Queue cannot be found?